### ISSUE TITLE
Implement MLP

### ISSUE THREAD

**Issue by dwettstein** [dwettstein/pattern-recognition-2016#5]
Use the provided training set to train an MLP with one hidden layer. Apply the trained MLP
to classify the test set. Perform cross-validation with the following parameters:
- Optimize number of neurons in the hidden layer (typically in the range [10,100]).
- Optimize learning rates (typically in the range [0.1,1])
- Optimize number of training iterations. Plot a graph showing the error on the training
  set and the validation set, respectively, with respect to the training epochs.
- Perform the random initialization several times and choose the best network during cross-
  validation.

https://pypi.python.org/pypi/bob.learn.mlp

**Comment by dwettstein** [comment]
Use MLP implementation of scikit-learn:
https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/neural_network/multilayer_perceptron.py

**Comment by dwettstein** [comment]
Alternative: https://pypi.python.org/pypi/scikit-neuralnetwork

**Comment by dwettstein** [comment]
Done with 2f010ff173e2abc25fe4d28ff4e21b1c645e474e
